### PR TITLE
Feature/add alibaba dragonwell

### PR DESCRIPTION
**Pull request by dzmitryastrouski**

### CODE REVIEW

**Comment by pavelpikta** [comment]
Issue #100 will be fixed in another PR

File: tasks/Linux/fetch/dragonwell8-fallback.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  fail:
+    msg: 'Dragonwell version {{ java_major_version }}.{{ java_minor_version }} not found'
+  when: response.status == 404
+
+# Transform Release name to Asset name
+# Release name sample: Alibaba Dragonwell 8.1.1-GA
+# Asset name sample: Alibaba_Dragonwell_8.1.1-GA_Linux_x64.tar.gz
+- name: Prepare release asset name
+  set_fact:
+    asset_name: "{{ (response.json.name | regex_replace(' ', '_')) + '_Linux_x64.tar.gz' }}"
```
Comment (tgadiev) [inline tasks/Linux/fetch/dragonwell8-fallback.yml]: Why do we need to set this strange variable?

File: tasks/Linux/fetch/dragonwell8-fallback.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+- name: Get release URL
+  set_fact:
+    release_url: "{{ response.json | json_query(\"assets[?name=='\" + asset_name + \"'].browser_download_url\") }}"
+
+- name: Download artifact from {{ release_url[0] }}
```
Comment (tgadiev) [inline tasks/Linux/fetch/dragonwell8-fallback.yml]: How about checksum?

Comment (dzmitryastrouski) [reply tasks/Linux/fetch/dragonwell8-fallback.yml]: GitHub API doesn't provide checksum for release assets. Check it yourself:
"assets": [
    {
      "url": "https://api.github.com/repos/alibaba/dragonwell8/releases/assets/13383833",
      "id": 13383833,
      "node_id": "MDEyOlJlbGVhc2VBc3NldDEzMzgzODMz",
      "name": "Alibaba_Dragonwell_8.0.0-GA_Linux_x64.tar.gz",
      "label": null,
      "uploader": {
        "login": "yfxhust",
        "id": 5487895,
        "node_id": "MDQ6VXNlcjU0ODc4OTU=",
        "avatar_url": "https://avatars1.githubusercontent.com/u/5487895?v=4",
      ...
      },
      "content_type": "application/x-gzip",
      "state": "uploaded",
      "size": 105990732,
      "download_count": 3164,
      "created_at": "2019-06-25T04:01:25Z",
      "updated_at": "2019-06-25T04:02:29Z",
      "browser_download_url": "https://github.com/alibaba/dragonwell8/releases/download/v8.0.0-GA/Alibaba_Dragonwell_8.0.0-GA_Linux_x64.tar.gz"
    },

**Pull request merged**